TRANSLATION_NOTES.asc

```diff
@@ -1,11 +1,11 @@
-== Translation Notes
+== Çeviri Notları
 
-After forking this repository to translate the work, this file is where the notes for coordinating the translation work would go.
-Things like standardizing on words and expressions so that the work is consistent or notes on how the contributing process is to be handled.
+İşi tercüme etmek için bu depoyu biçimlendirdikten sonra, bu dosya çeviri çalışmasını koordine etmek için notların gideceği yerdir.
+İşlerin tutarlı olmasını sağlamak için sözcük ve ifadeler üzerinde standartlaştırmak gibi şeyler; katkıda bulunan sürecin nasıl ele alınacağı üzerine notlar.
```

Comment:
Bu cümle biraz havada kalmış. Sonda yüklem yokmuş gibi durduğu için anlaşılması oldukça güç. Büyük ihtimalle ilk cümleye olan bağlılığından ötürü. Başka bir şekilde ifade etmek mümkün olabilir mi acaba?